js/client-battle.js

```diff
@@ -952,13 +953,70 @@
 				var move = Tools.getMove(thing);
 				if (!move) return;
 				var basePower = move.basePower;
-				if (!basePower) basePower = '&mdash;';
+				var basePowerText = '';
+				var additionalInfo = '';
+				var yourActive = this.battle.yourSide.active;
+				var myPokemon = this.battle.mySide.pokemon;
+
+				// Check if there are more than one active Pokémon to check for multiple possible BPs.
+				if (yourActive.length > 1) {
+					// We check if there is a difference in base powers to note it.
+					// Otherwise, it is just shown as in singles.
+					// The trick is that we need to calculate it first for each Pokémon to see if it changes.
+					var previousBasepower = false;
+					var difference = false;
+					var basePowers = [];
+					for (var i = 0; i < yourActive.length; i++) {
+						basePower = this.getMoveBasePower(move, this.battle.mySide.active[this.choice.choices.length], yourActive[i]);
+						if (previousBasepower === false) previousBasepower = basePower;
+						if (previousBasepower !== basePower) difference = true;
+						if (!basePower) basePower = '&mdash;';
+						basePowers.push('Base power for ' + yourActive[i].name + ': ' + basePower);
+					}
+					if (difference) {
+						basePowerText = '<p>' + basePowers.join('<br />') + '</p>';
+					}
+					// Falls through to not to repeat code on showing the base power.
+				}
+				if (!basePowerText) {
+					basePower = basePower || this.getMoveBasePower(move, myPokemon[this.choice.choices.length], yourActive[0]);
```

Comment:
@urkerab, it looks like this is the correct formulation. Can you make the change and revert dd760f1?